CIP-0028/README.md

```diff
@@ -0,0 +1,293 @@
+---
+CIP: 28
+Title: Protocol Parameters (Alonzo)
+Authors: Kevin Hammond <kevin.hammond@iohk.io>
+Status: Draft
+Type: Informational
```

Comment:
Shouldn't this be a standard CIP, considering these are parameters that all node implementation should be aware of?

Reply:
I feel the distinction to limit expectations: as understood this is meant as a "snapshot" CIP (and therefore might best sit as "Informational"). Current conversation around the changed block size since this PR was created is reflecting this. 
This is an ongoing conversation really.